schema.sql

```diff
@@ -555,6 +570,7 @@ CREATE OR REPLACE ACTION remove_credential($id UUID) PUBLIC {
     DELETE FROM access_grants WHERE data_id = $id;
 };
 
+-- @generator.description "Share a credential with another idOS user"
```

Comment:
```suggestion
-- @generator.description "Share a credential with creating AG"
```
It is not needed to be idOS user to retrieve a credential.